solafune_tools/community_tools/README.md

```diff
@@ -0,0 +1,33 @@
+# Community Tools Collection
+
+## Descriptions
+
+Tools repository where solafune developers and solafune hackers can share their tools on satellite data-related processing tools.
```

Comment:
@fulankun1412 i guess just stating "Solafune hackers" is ok for this part "solafune developers and solafune hackers"

Reply:
@shimokuracg Should we use Solafune's hackers or just Solafune hackers?

Reply:
If so, I will change the front README too because contains "Solafune's hackers" and do the solafune's developers need to change to "Solafune developers"?